internal/pkg/cli/env_init.go

```diff
@@ -368,6 +374,9 @@ func (o *initEnvOpts) validateCustomizedResources() error {
 	if (o.importVPC.isSet() || o.adjustVPC.isSet()) && o.defaultConfig {
 		return fmt.Errorf("cannot import or configure vpc if --%s is set", defaultConfigFlag)
 	}
+	if o.internalALBSubnets != nil && o.adjustVPC.isSet() || o.defaultConfig {
```

Comment:
I think this conditional will evaluate to true even when the user _only_ specifies the default config and no internal internal alb subnets. I believe we want parentheses here:
```suggestion
	if o.internalALBSubnets != nil && (o.adjustVPC.isSet() || o.defaultConfig) {
```